app/lib/package/search_service.dart

```diff
@@ -75,48 +74,46 @@ class SearchService {
 
   Future close() async {}
 
+  /// Returns the [PackageView] instance for [package] on its latest stable version.
+  Future<PackageView> getPackageView(String package) async {
```

Comment:
Should we consider making this `getPackageViews(List<String> packages)` to ensure that can still do batch loading?

Reply:
We do have and user `getPackageViews`, but not in a batching mode (yet, see TODO comment below). I'm making this private so that we don't encourage its use.